### PR TITLE
fix(field): remove nan fields from output

### LINKED ISSUE #693: bug: supply assistance & confirmed locations appears as "nan" in address information
## Bug Definition
In address information section the expression 'nan' appears . This bug is happening mostly in the 'Confirmed' , 'Supply Assistance' sections. (There are also a few error for some locations in the earthquake-affected section.)

-- Please add discord contact information
** discord username: @notDepresseDeveloper#0919 **

## Bug environment
Describe the environment produces the bug. For example:
afetharita.com

## Describe how you are producing the bug step by step

1. Go to afetharita.com 
2. Choose one of the 'Supply Assistance' or 'Confirmed' options from right panel
3. View the details of any location.
4. Bug appears

## Expected Behaviour
A clear and short text to decribe the expected behaviour.
We expect a statement other than help details not to be displayed, but to be displayed correctly.

## Screen shots
image

## Desktop Information
- Operating system: Win11
- Browser: Chrome
- Browser Version: 109.0.5414.120

### PR DESCRIPTION
**Pull request by Kayra1**
# Description

Some points on the map have a bold red nan written to their description. This is because some fields are nullable. Remove those fields from the final render to make things look less buggy.

**discord username: Kayra#1872**

**closes #693 **

This is a single line fix.

## Things to check before creating a PR
- [X] I have inspected my topic, checked.
- [X] If it is a core feature, executed detailed tests.

## Creating PR rules
- [X] PR must be created for an issue with approved tag. Otherwise PR will be rejected.
- [X] Relevant issue number: The issue number related to PR must be attached to head of PR header, after prefix after prefix # must be attached in parenthesis. A header like this could be used "prefix(#issue_number): PR header" .
- [X] A descriptive and understandable title: The PR title should clearly describe the nature and purpose of the changes. The PR title should be the first thing displayed when the PR is opened. And it should follow the semantic commit rules. For example, a title like "docs(#issueId): Add README.md" can be used.
- [X] Related file selection: Only relevant files should be touched and no other files should be affected.
- [X] Format and Lint Suitability : The code should be made in accordance with a certain format standard and examined according to the lint rules.
- [X] Clean commit history: The commits where the changes are made should be clear and organized.
- [X] Opening PR when job is completed: PR should be opened when job is completed and sent for review by other team members.

## Changes

- [ ] A new feature (a change that adds a new feature, not a breaking change)
- [X] A new refactor (a change that is not a breaking change, that improves the readability or performance of the code)
- [ ] A breaking change
- [ ] Documentation change

# How were these changes tested?

Checked the same points on the staging environment.

**Test Configuration**:

* Firmware version:
* Hardware:
* Toolchain:
* SDK:

### CODE REVIEW

**Comment by eraygundogmus** [comment]
please can you also/mainly help us on product management? you got more great knowledge about both frontend and backend. it would be great if you could! @Kayra1

**Comment by Kayra1** [comment]
Of course @eraygundogmus, you can reach me on discord any time my DM's are open

**Comment by eraygundogmus** [comment]
you can reach out any moderator and ask for afetharita-product-lead permission by showing this pr as proof

File: components/UI/Drawer/components/channels/FeedContent.tsx [hunk 1]
```diff
@@ -33,7 +33,9 @@ const FeedContent = ({ content }: Props) => {
     ? content.channel!
     : "generic";
 
-  // @ts-ignore: "content" parametresini tüm channel tipleriyle eşlemeye çalışıyor. Şimdilik ignore bırakıldı
+    // @ts-ignore
+    content.filter(e=>e)
+    // @ts-ignore: "content" parametresini tüm channel tipleriyle eşlemeye çalışıyor. Şimdilik ignore bırakıldı
```
Comment (cdagli) [inline components/UI/Drawer/components/channels/FeedContent.tsx:38]: @Kayra1 Ben tam anlamadim burada tam ne yapiyoruz?

Comment (cdagli) [reply components/UI/Drawer/components/channels/FeedContent.tsx:38]: Bir de ornek 2 tane post id paylasabilir misin bunu test edebilecegimiz, bir nan olan bir olmayan mumkunse?

**Comment by Kayra1** [comment]
Will be fixed in a future, more comprehensive PR.

**Pull request closed without merging**